packages/manager/modules/bm-server-components/src/support/template.html

```diff
@@ -6,23 +6,36 @@
     <!-- level -->
     <oui-tile-definition term="{{:: 'server_support_level' | translate }}">
         <oui-tile-description>
-            <span data-ng-bind=":: $ctrl.getFormatedSupportLevel()"></span>
+            <span
+                data-ng-if="!$ctrl.isHidden"
+                data-ng-bind=":: $ctrl.getFormatedSupportLevel()"
+            ></span>
+            <span
+                data-ng-if="$ctrl.isHidden"
+                data-translate="server_support_hidden"
+            ></span>
```

Comment:
**niptick** : To ease the use of IAM management. You can create a small wrapper component `checkIam`
```
<span data-ng-transclude data-ng-if="!$ctrl.isHidden"></span>
 <span
                data-ng-if="$ctrl.isHidden"
                data-translate="content_hidden"
            ></span>

```

```suggestion
            <check-iam data-is-hidden="$ctrl.isHidden">
            <span
                data-ng-bind=":: $ctrl.getFormatedSupportLevel()"
            ></span>
            </check-iam>
```